modules/edge-identity/src/identity.rs

```diff
@@ -91,6 +98,9 @@ decl_module! {
 			buf.extend_from_slice(&identity.encode());
 			let hash = T::Hashing::hash(&buf[..]);
 			ensure!(!<IdentityOf<T>>::exists(hash), "Identity already exists");
+			// Burn the registration bond amount
```

Comment:
Not a big deal but while you're making changes, I'd change this comment to reserve rather than burn.